### PR TITLE
Remove KDoc references

### LINKED ISSUE #41: Remove references to kdoc and --pdf
This is [already an issue on the legacy repo](https://github.com/kframework/k-legacy/issues/2310), but are there any plans to get KDoc working in K5, or at least to print out rules in that nice lego-block format, used e.g. on page 13 in the [SIMPLE paper](http://fsl.cs.illinois.edu/images/4/47/CS422-K-Overview.pdf)?

### PR DESCRIPTION
**Pull request by ehildenb**
Fixes #41 

The KDoc tool has been non-functional for some time. This removes it from the codebase and the documentation.

### CODE REVIEW

**Comment by radumereuta** [comment]
With this change, are latex attributes used anywhere else?
Shouldn't we remove those too?

**Comment by ehildenb** [comment]
They are still used by `--output latex` @radumereuta. And #1620 will make it actually use those attributes, so I don't want to remove them yet.

File: k-distribution/tutorial/1_k/1_lambda/lesson_9/README.md [hunk 1]
```diff
@@ -31,10 +30,9 @@ of the LAMBDA module, explaining how we define the syntax of this language.
 Note that we can simply use any Latex commands, including sections,
 subsections, mathematical symbols, and so on.
 
-Once we are done typing the annotations, we can `kompile lambda.k` with a
-document generation option, say `--pdf`, and take a look at the generated
-PDF document.  Other options are also possible, for example `--html` and
-`--latex`.  The latter is particularly useful when you want to incorporate
+Once we are done typing the annotations, we can `--output latex` to the
+invocation of `krun`, `kast`, and `kprove` and take a look at the generated
+PDF document. This is particularly useful when you want to incorporate
```
Comment (radumereuta) [inline k-distribution/tutorial/1_k/1_lambda/lesson_9/README.md]: I think this paragraph refers to generating a latex output of the definition. Having latex output for `krun`, `kast` and `kprove` is not really helpful in this case.

File: k-distribution/tutorial/1_k/4_imp++/lesson_8/README.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ to make them easier to read and print.
 Note that we rearrange a bit the semantics, to group the semantics of old
 IMP's constructs together, and separate it from the new IMP++'s semantics.
 
-Once we are done adding all the comments, we `kompile` with the documentation
-option `--pdf`, and then visualize the generated PDF document.
+Once we are done adding all the comments, we can `krun` with the documentation
+option `--output latex`, and then visualize the generated PDF document.
```
Comment (radumereuta) [inline k-distribution/tutorial/1_k/4_imp++/lesson_8/README.md]: Another reference to definition -> latex.

**Review by radumereuta — CHANGES_REQUESTED** [review]

File: k-distribution/tutorial/1_k/1_lambda/lesson_4/README.md [hunk 3]
```diff
@@ -36,11 +20,8 @@ In our case, we associate the attribute `latex(\lambda{#1}.{#2})` to the
...
-Let us now compile the new `lambda.k` using the option `--pdf`, and note how
-the latex attribute is being used in the produced output.
-
-Note that it is also possible to generates HTML documentation.  See
-`kdoc --help` for the exact command line options.
+Now, when you call `krun`, you can add the `--output latex` option to get a
+Latex AST of the output term. This option also works for `kast` and `kprove`.
 
 We will later see, in Lesson 9, that we can add arbitrarily complex Latex
 comments and headers to our language definitions, which give us maximum
```
Comment (radumereuta) [inline k-distribution/tutorial/1_k/1_lambda/lesson_4/README.md:24]: This paragraph still refers to using latex to format definitions.

**Review by radumereuta — CHANGES_REQUESTED** [review]

File: k-distribution/tutorial/1_k/1_lambda/lesson_4/README.md (hunk 3 above)
Comment (radumereuta) [reply k-distribution/tutorial/1_k/1_lambda/lesson_4/README.md:24]: I think this paragraph should be reworded to mention Markdown, or even deleted.

**Review by radumereuta — APPROVED** [review]
Looks fine for now.

File: k-distribution/tutorial/1_k/1_lambda/lesson_9/README.md [hunk 4]
```diff
@@ -9,9 +9,8 @@ to send it to others, etc.
 
 The K tool allows a literate programming style, where the executable
 language definition can be documented by means of annotations.  Some
-back-ends of the K tool, such as those enabled with the `--latex`, `--pdf` and
-the `--html` options to the `kompile` tool, know how to interpret such
-annotations.
+back-ends of the K tool, such as those enabled with the `--output latex`
```
Comment (dwightguth) [inline k-distribution/tutorial/1_k/1_lambda/lesson_9/README.md]: This seems to be missing a verb.

Comment (ehildenb) [reply k-distribution/tutorial/1_k/1_lambda/lesson_9/README.md]: addressed

**Review by dwightguth — APPROVED** [review]

**Pull request merged**